### DISCUSSION TITLE
Is there really no filter for features?

### DISCUSSION THREAD

**Discussion by chandelsboliger** [uber/react-map-gl#1575] [General]
I have looked everywhere and can't find a single site using react map gl where you are able to filter the features.

What i currently have is just a bunch of clusters with some unclustered points inside, where i want to filter those based on the GeoJSON data they have.

const geojson = { type: "FeatureCollection", crs: { type: "name", properties: { name: "urn:ogc:def:crs:OGC:1.3:CRS84" }, }, features: [ { type: "Feature", properties: { id: "cph1", mag: 2.3, time: 1507425650893, felt: null, tsunami: 0, }, geometry: { type: "Point", coordinates: [12.5, 55.67, 0.0], }, }, ], };

<Source id="apartments" type="geojson" data={geojson} cluster={true} clusterMaxZoom={14} clusterRadius={50} > <Layer {...clusterLayer} /> <Layer {...clusterCountLayer} /> <Layer {...unclusteredPointLayer} /> </Source>

**Comment by Pessimistress** [comment]
http://visgl.github.io/react-map-gl/examples/filter
